### DISCUSSION TITLE
Secret Levels

### DISCUSSION THREAD

**Discussion by CorridorWanterer** [nroutasuo/level13#305] [Ideas]
Dear nroutasuo,

I thought about making possible hidden sections or levels inside of areas. I had a thought about maybe there being some areas that you cannot initially access. Still, it is possible due to elevators or staircases that need to be repaired to access this little area with possibly some unexplored buildings or loot. For example, maybe there is a hidden level on level 13, but I need to access it by going into level 14 and exploring to find an elevator that will lead me to a small area on level 13, but would otherwise be inaccessible if I had not gone to level 14 and found the elevator. Or possibly when scavenging you find a hidden area. Maybe even hidden areas could be between levels, like a level 13 1/2, or level 13-12.

**Comment by nroutasuo** [comment]
Sounds fun! I'll consider it the next time I touch the world generator.
